src/components/FromSTIX/ChangeVersion.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import {Dropdown} from "carbon-components-react";
+import {changeStixVersion} from "../../store/actions/from_stix";
+
+import stixLangV2 from "../../global/stixLangV2";
+import stixLangV2_1 from "../../global/stixLangV2_1";
+import "./ChangeVersion.css"
+
+const itemstest = [
+    {
+        id: 0,
+        label: 'STIX version 2.0',
+    },
+    {
+        id: 1,
+        label: 'STIX version 2.1',
+
+    }
+];
+
+function title(){
+    return<div>
+        <ul className="change-title">
+            <li>Please select a STIX version</li>
+            <li>The default version is 2.0</li>
+        </ul>
+    </div>
+
+}
+const ChangeVersion = (props) => {
+    const dispatch = useDispatch();
+    const stixVersion = useSelector((state) => state.fromStix.stixFields);
+    return (
+        <div>
+        <Dropdown
```

Comment:
can add default selected value to dropdown, read about Dropdown component doc https://react.carbondesignsystem.com/?path=/story/components-dropdown--default